extendr-api/src/robj/mod.rs

```diff
@@ -770,6 +770,7 @@ macro_rules! make_typed_slice {
     }
 }
 
+make_typed_slice!(Rbool, INTEGER, LGLSXP);
 make_typed_slice!(Bool, INTEGER, LGLSXP);
```

Comment:
What would be the strategy for deprecating `Bool`?

Reply:
Lets keep it for a while, but stop documenting it.